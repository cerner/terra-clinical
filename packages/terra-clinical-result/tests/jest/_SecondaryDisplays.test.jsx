import React from 'react';
import SecondaryDisplays from '../../src/_SecondaryDisplays';

describe('SecondaryDisplays', () => {
  it('should render a default SecondaryDisplays', () => {
    const result = shallow(<SecondaryDisplays />);
    expect(result).toMatchSnapshot();
  });

  it('should render a SecondaryDisplays with concept display', () => {
    const result = shallow(<SecondaryDisplays conceptDisplay="concept" />);
    expect(result).toMatchSnapshot();
  });

  it('should render a SecondaryDisplays with datetime displays', () => {
    const result = shallow(<SecondaryDisplays datetimeDisplay="datetime" />);
    expect(result).toMatchSnapshot();
  });

  it('should render a SecondaryDisplays with all displays', () => {
    const result = shallow(<SecondaryDisplays conceptDisplay="concept" datetimeDisplay="datetime" />);
    expect(result).toMatchSnapshot();
  });
});
