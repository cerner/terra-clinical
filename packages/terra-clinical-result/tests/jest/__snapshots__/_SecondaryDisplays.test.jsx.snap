// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondaryDisplays should render a SecondaryDisplays with all displays 1`] = `
<Fragment>
  <div
    className="concept-display"
  >
    concept
  </div>
  <div
    className="datetime-display"
  >
    datetime
  </div>
</Fragment>
`;

exports[`SecondaryDisplays should render a SecondaryDisplays with concept display 1`] = `
<Fragment>
  <div
    className="concept-display"
  >
    concept
  </div>
</Fragment>
`;

exports[`SecondaryDisplays should render a SecondaryDisplays with datetime displays 1`] = `
<Fragment>
  <div
    className="datetime-display"
  >
    datetime
  </div>
</Fragment>
`;

exports[`SecondaryDisplays should render a default SecondaryDisplays 1`] = `<Fragment />`;
